nav2_controller/src/nav2_controller.cpp

```diff
@@ -37,11 +37,7 @@ ControllerServer::ControllerServer()
   RCLCPP_INFO(get_logger(), "Creating controller server");
 
   declare_parameter("controller_frequency", 20.0);
-  std::vector<std::string> default_id, default_type;
-  default_type.emplace_back("dwb_core::DWBLocalPlanner");
-  default_id.emplace_back("FollowPath");
-  declare_parameter("controller_plugin_ids", default_id);
-  declare_parameter("controller_plugin_types", default_type);
+  declare_parameter("controller_plugins");
```

Comment:
We still want to declare a default of DWB

Reply:
You could declare `controller_plugins` as `FollowPath` and `controller_plugins.plugin` as `DWB`.